config/examples/Elegoo/Neptune-2/Configuration.h

```diff
@@ -1627,7 +1655,7 @@
 //#define AUTO_BED_LEVELING_3POINT
 //#define AUTO_BED_LEVELING_LINEAR
 //#define AUTO_BED_LEVELING_UBL
-#if ENABLED(HAS_BLTOUCH)
+#if HAS_BLTOUCH
   #define AUTO_BED_LEVELING_BILINEAR
 #else
   #define MESH_BED_LEVELING
```

Comment:
Why not UBL for both? It's better to leave leveling types all disabled, because it's mostly a matter of taste.

Reply:
@EvilGremlin ,I see your point. I got this idea from looking at config for a machine with a similar board setup. [Two Trees - BlueR V3](https://github.com/just-trey/Configurations/blob/import-2.0.x/config/examples/Two%20Trees/BlueR/BlueR%20V3). My opinion is we leave it at is or leave leveling types all disabled. If someone wants to use UBL they would understand this enough to make the changes. Thoughts?